tests/sqlparser_postgres.rs

```diff
@@ -2507,6 +2520,7 @@ fn parse_array_subquery_expr() {
                     op: SetOperator::Union,
                     set_quantifier: SetQuantifier::None,
                     left: Box::new(SetExpr::Select(Box::new(Select {
+                        select_token: TokenWithLocation::wrap(Token::make_keyword("SELECT")).into(),
```

Comment:
To make places like tests that don't care about the token leaner, could we add a helper function like `AttachedToken::empty()` that returns a dummy token?

Reply:
I was sceptical about this, but it actually works pretty well 😅 
Not sure what Token to pick as a dummy token, so I went with `Token::EOF`

Reply:
Yeah EOF seems like natural/reasonable thanks!